skimage/morphology/isotropic.py

```diff
@@ -0,0 +1,133 @@
+"""
+Binary morphological operations
+"""
+import numpy as np
+from scipy import ndimage as ndi
+
+
+def isotropic_erosion(image, radius, out=None):
```

Comment:
Another enhancement that may be desirable to add here is a `spacing=None` keyword argument to allow specifying the pixel spacings used in the distance transform.  

I suggest `spacing` in the user-facing API for consistency with other scikit-image functions, but the argument to `distance_transform_edt` is called `sampling`, so the only change needed to support this is using:

```py
dist = ndi.distance_transform_edt(image, sampling=spacing)
```

This argument would allow getting a radius that is uniform in physical units for images with resolution that varies across axes. (It could, alternatively  be used to manipulate the distances giving an ellipsoidal erosion footprint, but I am less sure there is a use case for that).